### PR TITLE
Release 1.31.0

### PR DESCRIPTION
**Pull request by carlosalberto**
March 2024 release.

Changes to double review:

* #3858 
* #3887

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,41 +7,82 @@ release.
...
 ### Metrics
 
+- Clarify metric point.
+  ([#3906](https://github.com/open-telemetry/opentelemetry-specification/pull/3906))
 - Remove implementation specific specification from metric API.
   ([#3890](https://github.com/open-telemetry/opentelemetry-specification/pull/3890))
 - Formalize the interaction between cardinality limit and overflow attribute.
   ([#3912](https://github.com/open-telemetry/opentelemetry-specification/pull/3912))
+- Rework metric signal overview.
+  ([#3916](https://github.com/open-telemetry/opentelemetry-specification/pull/3916))
```
Comment (reyang) [inline CHANGELOG.md:50]: This PR doesn't change any requirement from the spec (more like an editorial change), probably no need to have it in the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,41 +7,82 @@ release.
...
 
 ### Logs
 
+- Fix: remove `name` from LogRecord example in the File Exporter example.
+  ([#3879](https://github.com/open-telemetry/opentelemetry-specification/pull/3879))
```
Comment (reyang) [inline CHANGELOG.md:55]: This should be #3886?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,41 +7,82 @@ release.
...
+  ([#3801](https://github.com/open-telemetry/opentelemetry-specification/pull/3801))
+
 ### Traces
 
+- No changes.
+
 ### Metrics
 
+- Clarify metric point.
+  ([#3906](https://github.com/open-telemetry/opentelemetry-specification/pull/3906))
```
Comment (reyang) [inline CHANGELOG.md:44]: Remove this, it's an editorial change.

**Comment by carlosalberto** [comment]
So in general we don't add editorial fixes to the CHANGELOG but the two mentioned ones are not one or two-liners, but actual clarifications, so I have mixed feelings about them being included or not.

**Comment by MrAlias** [comment]
Can we include the following PRs in the release?

- https://github.com/open-telemetry/opentelemetry-specification/pull/3889
- https://github.com/open-telemetry/opentelemetry-specification/pull/3853

They both look ready to merge.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,41 +7,82 @@ release.
...
+
+### Compatibility
+
+### SDK Configuration
+
+### Common
+
+### Supplementary Guidelines
+
+## v1.31.0 (2024-03-06)
```
Comment (arminru) [inline CHANGELOG.md:30]:
```suggestion
## v1.31.0 (2024-03-07)
```
```suggestion
## v1.31.0 (2024-03-08)
```

File: CHANGELOG.md (hunk 1 above)
Comment (jmacd) [reply CHANGELOG.md:50]: Maybe we could have an "editorial" section of the changelog?

**Review by jmacd — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (carlosalberto) [reply CHANGELOG.md:30]: Hopefully today instead? :D

File: CHANGELOG.md (hunk 1 above)
Comment (cijothomas) [reply CHANGELOG.md:50]: > Maybe we could have an "editorial" section of the changelog?

Do we need that? There is git blame for anyone interested in entire history anyway.

**Comment by carlosalberto** [comment]
FYI @MrAlias #3889 was changed to draft as it needs more discussion, but #3853 was merged.

File: CHANGELOG.md (hunk 1 above)
Comment (carlosalberto) [reply CHANGELOG.md:50]: Let's remove it for now and decide on a case-by-case fashion.

**Review by pellared — APPROVED** [review]

**Comment by MrAlias** [comment]
> FYI @MrAlias #3889 was changed to draft as it needs more discussion, but #3853 was merged.

Right, sorry I should have said I understood that explicitly when I approved. Not blocking on getting that PR merged, all good :+1:

**Pull request merged**